photo_points/Repositories/IAdminReviewRepository.cs

```diff
@@ -4,12 +4,12 @@
 using System.Threading.Tasks;
 using photo_points.Models;
 
-namespace photo_points.Models
+namespace photo_points.Repositories
 {
     public interface IAdminReviewRepository
     {
         IQueryable<Capture> GetCaptures();
         Capture GetCapture(long captureId);
-        IEnumerable<Capture> GetAllUnapproved();
+        void SaveChanges(Capture capt);
```

Comment:
Hi Eric, What does this line do?

Reply:
That line would save any changes made to a capture in the repository. Sara was working on this method.